indexer/database/blocks.go

```diff
@@ -174,12 +175,14 @@ func (db *blocksDB) LatestObservedEpoch(fromL1Height *big.Int, maxL1Range uint64
 		fromL1Height = bigint.Zero
 	}
 
-	// Lower Bound (the default `fromTimestamp = 0` suffices genesis representation)
+	// Lower Bound (the default `fromTimestamp = l1_starting_heigh` (default=0) suffices genesis representation)
 	if fromL1Height.BitLen() > 0 {
 		var header L1BlockHeader
 		result := db.gorm.Where("number = ?", fromL1Height).Take(&header)
 		if result.Error != nil {
 			if errors.Is(result.Error, gorm.ErrRecordNotFound) {
+				log.Warn("Could not fetch latest L1 block header in bridge processor", "number", fromL1Height,
```

Comment:
rm these log lines. You can push these into the bridge processer though when it calls `LatestObservedEpoch`

Reply:
[a86737d](https://github.com/ethereum-optimism/optimism/pull/7649/commits/a86737dfd3ef8961c485586d7b9f2b790a67f460)